README.md

```diff
@@ -1 +1,132 @@
-# Paketo Composer Install Buildpack
+# PHP Composer Distribution Cloud Native Buildpack
+
+This buildpack runs the `composer install` command [composer](https://getcomposer.org/) to download project dependencies.
+It requires both `composer` and `php` on the path (see [requires](#requires)).
+
+A usage example can be found in the
+[`samples` repository under the `php/composer` directory](https://github.com/paketo-buildpacks/samples/tree/main/php/composer).
+
+## Detection
+
+Will add these requires/provisions to the build plan if and only if a `composer.json` file is found.
```

Comment:
Should we add a note that a `composer.lock` is preferred to be there as well?

Reply:
done, although it's in the `## Build` section.